Multisig/Features/Connect to Web/UI/Details/WebConnectionDetailsViewController.swift

```diff
@@ -167,7 +167,7 @@ class WebConnectionDetailsViewController: UITableViewController, WebConnectionOb
         case .description:
             let cell = contentCell()
             cell.setText("Description")
-            let text = peer.description ?? "Unknown"
+            let text = peer.description ?? peer.name
```

Comment:
is the reason that the description shows empty the null text or is that the text string itself is empty?

Reply:
right, the second one